app/model/sdl/VehicleInfoModel.js

```diff
@@ -331,6 +332,16 @@ SDL.SDLVehicleInfoModel = Em.Object.create(
          'width': 800,
          'height': 480
       },
+      'seatStatus':{
+        'seatLocation': true,
+        'conditionActive':true
+      },
+      'seatOccupancy':{
+        'seatsOccupied'[]:true,
+        'seatsOccupied'.length <100,
+        'seatsBelted'[]:true,
+        seatsBelted.length <100
+      },
```

Comment:
@Ypostolov wtf is this? Just declare the arrays
```suggestion
      'seatOccupancy':{
        'seatsOccupied': [],
        'seatsBelted': []
      },
```

Reply:
@AKalinich-Luxoft 
Fixed in 6db411a .